### PR TITLE
Added imunities and traits to Morille Mortelle

### PR DESCRIPTION
**Pull request by dwn134**

### CODE REVIEW

File: scripts/zones/Crawlers_Nest_[S]/mobs/Morille_Mortelle.lua [hunk 1]
```diff
@@ -26,3 +38,26 @@ function onMobDespawn(mob)
...
+    if (math.random(0,99) >= chance or resist <= 0.5) then
+        return 0,0,0;
+    else
+        local duration = 30;
+        if (mob:getMainLvl() > player:getMainLvl()) then
+            duration = duration + (mob:getMainLvl() - player:getMainLvl())
+        end
+        duration = utils.clamp(duration,1,45);
+        duration = duration * resist;
+        if (not player:hasStatusEffect(EFFECT_PLAGUE)) then
```
Comment (TeoTwawki) [inline scripts/zones/Crawlers_Nest_[S]/mobs/Morille_Mortelle.lua]: See: https://github.com/DarkstarProject/darkstar/pull/3917#discussion_r123145549

I will wait to see if any other project members comment for a few days.

**Comment by TeoTwawki** [comment]
@dwn134 you could have just force pushed over your old fork to accomplish this.

**Comment by dwn134** [comment]
I went to rebash like you said but it only showed the nm changes and not
the pup files i copied over. Ill keep my personal server seprate from my
forks from now on to not have this happen again

**Comment by TeoTwawki** [comment]
let us help you next time, these mistakes happen a lot to newer folks

**Comment by dwn134** [comment]
Yea i thought to just start over would be less work for everyone. My mistake

File: scripts/zones/Crawlers_Nest_[S]/mobs/Morille_Mortelle.lua (hunk 1 above)
Comment (TeoTwawki) [inline scripts/zones/Crawlers_Nest_[S]/mobs/Morille_Mortelle.lua]: 12 days of silence = acceptance

**Pull request merged**